<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>训练界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;新宋体&quot;;</string>
      </property>
      <property name="title">
       <string>控制窗口</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,3">
         <item>
          <widget class="QPushButton" name="QpushButton_background">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>切换背景</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="QComboBox_background">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,3">
         <item>
          <widget class="QPushButton" name="QPushButton_weight">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>选择初始权重</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_weight">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,0,0,0">
         <item>
          <widget class="QPushButton" name="QPushButton_data">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;
background:rgb(85, 255, 255)</string>
           </property>
           <property name="text">
            <string>样本统计</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="QPushButton_data_supply">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;
background:rgb(85, 255, 255)</string>
           </property>
           <property name="text">
            <string>补充样本</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="QPushButton_opentensorboard">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;
background:rgb(133, 255, 251)</string>
           </property>
           <property name="text">
            <string>打开Tensorboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;新宋体&quot;;
</string>
      </property>
      <property name="title">
       <string>训练参数</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,3">
         <item>
          <widget class="QPushButton" name="pushButton_suggest">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>使用推荐配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="QComboBox_suggest">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0,0,0,0">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string>批量大小：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_batch_size">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>n次无提升停止训练：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_patience">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string>线程数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_worker">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string>训练次数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_epoch">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
           </property>
           <property name="text">
            <string>损失函数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="QComboBox_optimize">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>新宋体</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>SGD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Adam</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Adamw</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stop">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:rgb(255, 165, 166);
font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>提前停止训练</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_continue">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:rgb(203, 255, 197);font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>继续上次训练</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>开始训练</string>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_final">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;新宋体&quot;;</string>
           </property>
           <property name="text">
            <string>退出训练</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
